### PR TITLE
Pangolin visualization

### PR DESCRIPTION
**Pull request by ikryukov**

### CODE REVIEW

File: include/core.h [hunk 1]
```diff
@@ -65,8 +65,8 @@ void trackingFrame2Frame(const cv::Mat& projMatrl,
                          std::vector<cv::Point2f>& pointsLeft_t0,
                          std::vector<cv::Point2f>& pointsLeft_t1,
                          cv::Mat& points3D_t0,
-                         cv::Mat& rotation,
-                         cv::Mat& translation);
+                         cv::Matx<double, 3, 3>& rotation,
```
Comment (ikryukov) [inline include/core.h]: why double?

File: include/core.h [hunk 2]
```diff
@@ -85,7 +85,7 @@ void distinguishNewPoints(std::vector<cv::Point2f>& newPoints,
 void integrateOdometryStereo(int frame_i,
                              cv::Mat& rigid_body_transformation,
                              cv::Mat& frame_pose,
-                             const cv::Mat& rotation,
-                             const cv::Mat& translation_stereo);
+                             const cv::Matx<double, 3, 3>& rotation,
```
Comment (ikryukov) [inline include/core.h]: maybe it is better to use 'using' or typedef for this?

File: src/drawer.cpp [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        }
+    }
+}
+
+void Drawer::run(Drawer* drawer) {
+    pangolin::CreateWindowAndBind("VO", 640, 480);
+    glEnable(GL_DEPTH_TEST);
+
+    pangolin::OpenGlRenderState s_cam(
+        pangolin::ProjectionMatrix(1024, 768, 500, 500, 512, 389, 0.1, 1000),
```
Comment (ikryukov) [inline src/drawer.cpp:102]: move to const or parameters?

File: include/core.h (hunk 1 above)
Comment (alexeyalekzz) [reply include/core.h]: It was declared as double before
`cv::Mat rotation = cv::Mat::eye(3, 3, CV_64F);`
`cv::Mat translation_stereo = cv::Mat::zeros(3, 1, CV_64F);`

**Pull request merged**